cirq/experiments/random_quantum_circuit_generation.py

```diff
@@ -154,6 +154,69 @@ def __repr__(self) -> str:
 )
 
 
+def random_rotations_between_two_qubit_circuit(
```

Comment:
wow, this method name is tough. I came up with even longer ones :)
- `random_rotation_layers_interleaved_with_two_qubit_gate_layers_circuit`
- `random_layered_interleaved_single_and_two_qubit_gates_circuit`
- `alternating_random_single_and_two_qubit_layers_circuit`

Reply:
(optional)

Reply:
the current name was by analogy to the existing function `random_rotations_between_grid_interaction_layers_circuit`. Let's circle back on this name later. End users likely won't be using this directly, and I think the analogy to the other function is helpful for developers. This notebook shows the details that will be abstracted away in follow on PRs.

Reply:
sgtm!